bundles/org.openhab.binding.km200/src/main/resources/ESH-INF/thing/systemStates.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="km200"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="systemStates" listed="false">
```

Comment:
At first sight a thing does not seem like the proper abstraction level, can you give some more information why it became a thing? Should it just be a (dynamic) channel on an existing thing. Thanks for your time and efforts.

Reply:
Hi,

all things in this bindig are dynamicaly created. This is a new one in same way. We need this file because the dynamic creation is not working without.

Greetings
Markus

Reply:
@martinvw , Hi, this is still open. Do you need something else? Greetings

Reply:
Thanks for the reminder